src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx

```diff
@@ -56,8 +53,13 @@ const Contacts = ({ data }) => {
           <ul className="vads-u-margin-y--0">
             <li>Ask a staff member at your next appointment, or</li>
             <li>
-              Call us at <HelpDeskContact testId="help-desk" />. We’re here
-              Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
+              Call your VA health facility.{' '}
+              <href
+                data-testid="find-locations-additional-info-link"
+                to="https://www.va.gov/find-locations/"
```

Comment:
It's surprising that our linters missed an `<href />` tag.

`<a href="/find-locations" ... />` would work, and also point to the appropriate hosting environment (e.g. - dev.va.gov, staging.va.gov)

Reply:
thank you!